src/pyodide/python-entrypoint-helper.ts

```diff
@@ -232,16 +232,28 @@ async function initPyInstance(
   return res as PyModule;
 }
 
+// https://developers.cloudflare.com/workers/runtime-apis/rpc/reserved-methods/
+const SPECIAL_HANDLER_NAMES = ['fetch', 'connect'];
+const SPECIAL_DO_HANDLER_NAMES = [
+  'alarm',
+  'webSocketMessage',
+  'webSocketClose',
+  'webSocketError',
```

Comment:
should we also translate these to snake case e.g., `websocket_message`?

Reply:
Please keep it same as JS, even if it is a tiny bit less python idiomatic

Reply:
or special case both and fail startup if both are defined